dev-workflow-ui/webContent/view/statistics.xhtml

```diff
@@ -0,0 +1,26 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://xmlns.jcp.org/jsf/core"
+  xmlns:h="http://xmlns.jcp.org/jsf/html" xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+  xmlns:ic="http://ivyteam.ch/jsf/component" xmlns:p="http://primefaces.org/ui"
+  xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+  <ui:composition template="../includes/template/template.xhtml">
+    <ui:define name="content">
+      <f:event type="preRenderView" listener="#{loginIvyDevWfBean.redirectIfNotAdmin}" />
+      <div class="card">
+        <div class="flex align-items-center mb-3">
+          <i class="pr-3 si si-network-browser card-title-icon" />
+          <h2 class="m-0">Statistics</h2>
+        </div>
+      </div>
```

Comment:
IMO this looks not good, to have the title separated in its own card. If not better possible I would remove it completely.